configuration/parse.go

```diff
@@ -0,0 +1,80 @@
+package configuration
+
+import (
+	"bufio"
+	"errors"
+	"flag"
+	"fmt"
+	"github.com/sonatype-nexus-community/nancy/types"
+	"os"
+	"strings"
+)
+
+type Configuration struct {
+	NoColor bool
+	Quiet   bool
+	Version bool
+	CveList types.CveListFlag
+	Path    string
+}
+
+func Parse(args []string) (Configuration, error) {
+	config := Configuration{}
+	var excludeVulnerabilityFilePath string
+
+	flag.BoolVar(&config.NoColor, "noColor", false, "indicate output should not be colorized")
+	flag.BoolVar(&config.Quiet, "quiet", false, "indicate output should contain only packages with vulnerabilities")
+	flag.BoolVar(&config.Version, "version", false, "prints current nancy version")
+	flag.Var(&config.CveList, "exclude-vulnerability", "Comma separated list of CVEs to exclude")
+	flag.StringVar(&excludeVulnerabilityFilePath, "exclude-vulnerability-file", "./exclude_vulnerabilities", "Path to a file containing newline separated CVEs to be excluded")
```

Comment:
I feel like we could make this file have a default location and get picked up automagickally, wdyt?

Reply:
`./exclude_vulnerabilities` is the default path. So wherever you are running nancy from. I dont know what a good default would be but im up for suggestions. It's easy enough to just change that default value. 

Are you thinking something like `~/.sonatype/cve-exclude_vulnerabilities`??

Reply:
I am kinda thinking in the root of the project, like `.nancy-ignore` or something like that, and if you already have a sane default that's great.

Reply:
.nancy-ignore feels reasonable. I'll switch it to that. Root of the project might be difficult since I don't have any context to where people are running it from thus why I went with cwd